### PR TITLE
Implement the Angular position capture

### PR DESCRIPTION
**Pull request by Juudyyt**
Translate from AngularPosition-Buff-Cont.c to python API

### CODE REVIEW

**Comment by Juudyyt** [comment]
add new file translation

File: angularcapture.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import nidaqmx
+from nidaqmx.stream_readers import CounterReader
+from nidaqmx.constants import EncoderType, EncoderZIndexPhase, AngleUnits
+import numpy as np
+import click
+
+
+@click.command()
+@click.argument('samples', type=click.INT)
+@click.argument('encodertype')
```
Comment (rhomspuron) [inline angularcapture.py]: https://click.palletsprojects.com/en/7.x/options/
Could you change to use click.Choice?

File: angularcapture.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    argum=check_arguments(encodertype, encoderzindex, anlgeunits)
+
+    with nidaqmx.Task() as task:
+        # DAQmxCreateCIAngEncoderChan(taskHandle,"Dev1/ctr0","",DAQmx_Val_X4,0,0.0,DAQmx_Val_AHighBHigh,DAQmx_Val_Degrees,24,0.0,"")
+        # task.ci_channels.add_ci_ang_encoder_chan(counter, "", EncoderType.X_4, False, 0, EncoderZIndexPhase.AHIGH_BHIGH, AngleUnits.DEGREES, 24, 0.0, "")
+        task.ci_channels.add_ci_ang_encoder_chan(counter, "", argum[0], False, 0, argum[1], argum[2], 24, 0.0, "")
+        # DAQmxCfgSampClkTiming(taskHandle,"/Dev1/PFI9",1000.0,DAQmx_Val_Rising,DAQmx_Val_ContSamps,1000)
+        task.timing.cfg_samp_clk_timing(1000.0, source_trigger, samps_per_chan=samples) # (source_trigger, Edge.RISING, AcquisitionType.FINITE, 1000)
+
```
Comment (rhomspuron) [inline angularcapture.py]: Could you use flake8 to check PEP8?
https://flake8.pycqa.org/en/latest/

File: angularcapture.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    elif anlgeunits == "FROM_CUSTOM_SCALE":
+        argum.append(AngleUnits.FROM_CUSTOM_SCALE)
+    elif anlgeunits == "RADIANS":
+        argum.append(AngleUnits.RADIANS)
+    elif anlgeunits == "TICKS":
+        argum.append(AngleUnits.TICKS)
+    else:
+        usage()
+
+    return argum
```
Comment (rhomspuron) [inline angularcapture.py]: You can remove it after to use click.Choice

**Review by rhomspuron — CHANGES_REQUESTED** [review]
Thanks for the contribution!

**Comment by Juudyyt** [comment]
code correction

**Comment by Juudyyt** [comment]
code correction

**Review by rhomspuron — APPROVED** [review]

**Comment by rhomspuron** [comment]
Thanks for the contribution @Juudyyt

**Pull request merged**